src/utils/data/animals.mock.json

```diff
@@ -1,89 +1,67 @@
 {
   "edges": [
+    { "source": "9d989e8d", "target": "f42dab68" },
```

Comment:
This formatting is so much better and sanity preserving! Thanks @MerelTheisenQB and/or `black` (I presume) 👏